patches/chromium/fix_crash_when_saving_edited_pdf_files.patch

```diff
@@ -0,0 +1,86 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Shelley Vohr <shelley.vohr@gmail.com>
+Date: Mon, 17 Jan 2022 23:47:54 +0100
+Subject: fix: crash when saving edited PDF files
+
+This commit fixes a crash that persists any time a user attempts to
+download an edited PDF. This was happening because the logic flow for
+downloading of any edited PDF triggers a call to
+chrome.fileSystem.chooseEntry, which we do not support and which
+therefore causes unmapped page access crashes.
+
+This patch can be removed should we choose to support chrome.fileSystem
+or support it enough to fix the crash.
+
+diff --git a/chrome/browser/resources/pdf/pdf_viewer.js b/chrome/browser/resources/pdf/pdf_viewer.js
+index 42407ef7c214bed1eb44165a87c6d0dc44f0ad7f..afc8b93eff9c8b325555b0a517936bdf760394b2 100644
+--- a/chrome/browser/resources/pdf/pdf_viewer.js
++++ b/chrome/browser/resources/pdf/pdf_viewer.js
+@@ -967,25 +967,12 @@ export class PDFViewerElement extends PDFViewerBaseElement {
+       dataArray = [result.dataToSave];
+     }
+ 
++    const a = document.createElement('a');
++    a.download = fileName;
+     const blob = new Blob(dataArray);
+-    const fileName = this.attachments_[index].name;
```

Comment:
Looking at the patched file, it does not appear that `fileName` is defined.

Reply:
ah oops i'll fix this - bad copy from the non-attachment section!

Reply:
done!